torch/csrc/jit/codegen/cuda/runtime/helpers.cu

```diff
@@ -279,3 +279,19 @@ template <>
 double pow<double>(double a, double b) {
   return ::pow(a, b);
 }
+
+float pow(float a, int b) {
+  return pow(a, (float)b);
+}
+
+float pow(double a, int b) {
```

Comment:
Shouldn't the return type be `double`?

Reply:
Should we just have
```
template<typename T0,
         typename T1,
         typename = std::enable_if_t<std::is_floating_point<T0>::value>,
         typename = std::enable_if_t<!std::is_same<T0, T1>::value>>
T0 pow(T0 a, T1 b) {
  return pow(a, (T0) b);
}
```

Reply:
Can we import type traits in nvrtc? Leaving as is for now but feel free to follow up on it.